### PR TITLE
remove submit_child_run method

### PR DESCRIPTION
**Pull request by diondrapeck**

### CODE REVIEW

File: R/run.R [hunk 1]
```diff
@@ -870,22 +870,6 @@ create_child_runs <- function(parent_run,
...
-#'
-#' @description
-#' Submit an experiment and return the active child run.
-#' @param parent_run The parent `Run` object.
-#' @param config The `RunConfig` object
-#' @param tags Tags to be added to the submitted run, e.g., {"tag": "value"}.
-#' @return A `Run` object.
-#' @export
-#' @md
-submit_child_run <- function(parent_run,
```
Comment (heemanshusuri) [inline R/run.R]: why remove the function? let's try to add support for child runs in estimators in Python SDK.

Comment (diondrapeck) [reply R/run.R]: okay, sounds good. In the meantime though, should we un-export it so users don't try using the estimator runconfig (like in the failed test) and get errors?

Comment (heemanshusuri) [reply R/run.R]: Adding @mx-iao

Comment (mx-iao) [reply R/run.R]: yeah, for now let's unexport the method

**Pull request merged**